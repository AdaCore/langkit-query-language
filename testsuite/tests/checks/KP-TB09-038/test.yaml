driver: 'checker'
rule_name: KP_TB09_038
project: 'prj.gpr'
