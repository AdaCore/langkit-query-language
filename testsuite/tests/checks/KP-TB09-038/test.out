p.ads:3:8: rule violation: possible occurrence of KP TB09-038
3 |   type Biased_T is range 1 .. 2 ** 6;      --  FLAG
  |        ^^^^^^^^

